src/frontend/src/components/public/PIASideNav/pia-form-sideNav-pages.ts

```diff
@@ -45,6 +46,26 @@ export const PiaFormSideNavPages = (
     }
   };
 
+  const enableReview = (): boolean => {
+    if (
+      pia?.hasAddedPiToDataElements === false &&
+      pia?.isNextStepsSeenForDelegatedFlow === true
+    ) {
+      if (isMPORole()) {
```

Comment:
I thought Brandon said we will get rid of this and start to use roleCheck function, @Bottle7  Can you confirm?

Reply:
That was the idea, yeah.